### DISCUSSION TITLE
Make the `News` links more evident.

### DISCUSSION THREAD

**Discussion by SirWumpus** [ioccc-src/temp-test-ioccc#2300] [Ideas]
I think the `News` menu should be a menu/nav bar below the title so as to be more prominent.  I found I had to search the HTML source to find where the FAQ link was.
![prominent_news](https://github.com/ioccc-src/temp-test-ioccc/assets/7441729/01604264-428b-481d-a4e3-9bdd4911d22c)

**Comment by lcn2** [comment]
We plan to rework the menu system before it is released.

The current menu set, titles and keywords are just placeholders put there by the web developer that the IOCCC is working with.

Your idea about using a FAQ heading is a good one. Links for things like "How to enter" or "Fixing IOCCC entries" would go under such as FAQ set.

Never fear, we will rework the menu system and welcome feedback such as what you give us @SirWumpus, before the Great Fork Merge happens. In the mean time we are focusing, with the web developer, on the function of the menus instead of the form at this stage.

## UPDATE

This issue has been added to the issue #2239 TODO list.  Thank you @SirWumpus for your feedback!
